packages/twenty-server/src/engine/metadata-modules/object-metadata/object-metadata.service.ts

```diff
@@ -341,7 +356,7 @@ export class ObjectMetadataService extends TypeOrmQueryService<ObjectMetadataEnt
     );
 
     createdObjectMetadata.fields.map(async (field, index) => {
-      if (field.name === 'id') {
+      if (field.name === 'id' || field.name === 'deletedAt') {
```

Comment:
I'm not sure we want to do that. When you display the table view with "soft-deleted" filter on, you probably want to know the date of soft deletion? 
cc @charlesBochet